ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewGroupManager.java

```diff
@@ -64,25 +64,36 @@ public int getChildCount(T parent) {
     return parent.getChildCount();
   }
 
+  public int getUnmarkedChildCount(T parent) {
```

Comment:
can you add documentation about what is the purpose of these new methods?

Also let's revisit the names, you are mixing View with Child, e.g. markView vs getMakedChildAt